xml/System.Composition.Hosting/ContainerConfiguration.xml

```diff
@@ -242,10 +242,10 @@
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="TPart">To be added.</typeparam>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TPart">The type to add.</typeparam>
+        <summary>Adds the specified generic type to the container.</summary>
```

Comment:
Since the generic type is an abstraction that's resolved by a specific type, it's best to phrase this as "Adds the specified type to the container."